src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -49,6 +49,18 @@ public boolean isValidViewer() {
     public boolean isTeam() {
         return this == TEAMS || this == OWN_TEAM;
     }
+    
+    public boolean isOwnTeamMembers() {
+        return this == OWN_TEAM_MEMBERS;
```

Comment:
Got it. This check should move out of `FeedbackParticipantType` then, since it's dependent on that specific context